app/views/media/index.html.erb

```diff
@@ -6,7 +6,7 @@
                       lecture_material_path(@lecture,
                                         params: { project: params[:project],
                                                   lecture_id: params[:lecture_id],
-                                                  reverse: !params[:reverse],
+                                                  reverse: (params[:reverse] != "true").to_s,
                                                   all: params[:all],
```

Comment:
This fixes an actual bug in the current implementation so I thought let's fix it here instead of opening a dedicated PR for this simple change.

Reply:
Ah good catch. You might want to just use `params[:reverse] != "true"` without `to_s`. Please also fix this for watchlists (search for `!params[:reve`).

Reply:
Fixed.

Reply:
Oops. I just realized that in the watchlists controller there is parameter sanitization:

````ruby
 params[:reverse] = params[:reverse] == "true"
```

That's why this change should not be done for watchlists. I changed it back for watchlists.